checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -158,9 +158,15 @@ def _check_suppression(self, record: Record, suppression: dict[str, Any]) -> boo
         elif type == 'Cves':
             if 'accountIds' not in suppression:
                 return False
-            if self.bc_integration.repo_id in suppression['accountIds'] and record.file_abs_path == suppression['cves'][0]['id'][1:]:
-                return any(record.vulnerability_details and record.vulnerability_details['id'] == cve['cve']
-                           for cve in suppression['cves'])
+            if self.bc_integration.repo_id in suppression['accountIds']:
+                repo_name = self.bc_integration.repo_id.replace('\\', '/').split('/')[-1]  # type: ignore
+                suppression_path = suppression['cves'][0]['id'].replace('\\', '/')
```

Comment:
I know you just moved it here, but do you know why we use only the index 0 of `suppression['cves']`?
in some cases (specifically in the UTs) there are more than 1 items in the list

Reply:
true, there can be more than one cve, but each suppression represent one file, so if we have few cves that were suppressed on the same file, the 'id' which is the file path will be the same for all of them